ABOUT.md

```diff
@@ -0,0 +1,51 @@
+
+# What is MineColonies?
+Here, we would like to give you an overview of what our mod is.
+
+### Creativity 
+MineColonies is a town building mod that allows you create your own thriving Colony within Minecraft. It tends to depict a real-life scenario by providing you with different tools in your chest to aid your gaming experience. Featuring many NPC workers such as Builders, Crafters, Farmers, Guards, Enchanters and Alchemists, MineColonies provides the ability to create a rich and unique Colony.
+
+### Governance
+In MineColonies, you take control of what goes on in your Colony. Have an adequate number of villagers who will live in the town with you. Create a division of labour, hire workers to help in developing the Colony while you maturely act as a Supervisor.
+
+### Security and Replenishment
+The growth and safety of the colony solely depends on your governmental skills. Barbarians attack the colony on few nights. Hire guards, assign them weapons and armour to aid in protecting the Colony. As a Colony owner, you cannot let your villagers go hungry. You can always get food from your chest to feed the hungry villagers.
```

Comment:
I would add/write: "Your Town deservers the best security you can have. Strategically add Guard Towers or Guard Barracks to protect your Town from the night mobs and those barbarians that will attack your Town in random nights. Give them the best weapons and armor your Town has to offer. Your Town needs to be fed well to work and protect it better. Keep a busy farmer to supply the baker and the cook at the restaurant, so no one goes hungry."